src/main/java/Task.java

```diff
@@ -0,0 +1,26 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
```

Comment:
Do we have to use the `this` keyword for isDone when it is not being overshadowed?